test/v2/client/test_client.go

```diff
@@ -914,3 +914,16 @@ func (c *TestClient) GetProxyWrapper() (*ProxyWrapper, error) {
 	}
 	return c.proxyWrapper, nil
 }
+
+func (c *TestClient) EstimateGasCheckDACert(
```

Comment:
```suggestion
func (c *TestClient) EstimateGasAndReportCheckDACert(
```

Reply:
done in [85e3469](https://github.com/Layr-Labs/eigenda/pull/1890/commits/85e3469d534a655ed598270c9e9f43fcbcafae39)